src/common/copilot/dataverseMetadata.ts

```diff
@@ -25,6 +25,7 @@ export async function getEntityColumns(entityName: string, orgUrl: string, apiTo
             headers: {
                 'Content-Type': "application/json",
                 Authorization: `Bearer ${apiToken}`,
+                "x-ms-user-agent": "Vscode-Desktop"
```

Comment:
This is a great idea.
The expected format of this string is the same as the [User-Agent header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent). Namely:
`x-ms-user-agent: <product> / <product-version> <comment>`

1. Let's add the version of the product too. It must be the version of the 'product'
1. For 'product', Should it actually be the name or id of the extension? If not, and we want it to be the VSCode product, can we set the product value to something we get from the VSCode SDK instead?
2. In the 'comment' part, is where extra information is usually set. e.g. the desktop vs web would be more appropriate there. e.g. `VSCode/1.88.1 (Desktop; other; another)`

Reply:
Done